docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+     - Stable
+     -
+   * - ``bindings-rxjava`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.6-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.7-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.8-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-dev-archive-java-proto``
+     - Alpha
+     -
+   * - **Node.js Ledger API Bindings**
+     -
+     -
+   * - ``@digital-asset/binings-js`` :doc:`Node.js library </app-dev/bindings-js>`
+     - Stable
+     - 
+   * - **JavaScript Client Libraries**
+     -
+     -
+   * - ``daml codegen js`` :doc:`CLI and generated code </app-dev/bindings-ts/daml2js>`
+     - Stable
+     -
+   * - ``@daml/types`` :doc:`library and its public API </app-dev/bindings-ts/daml-types>`
+     - Stable
+     -
+   * - ``@daml/ledger`` :doc:`library and its public API </app-dev/bindings-ts/daml-ledger>`
+     - Stable
+     -
+   * - ``@daml/react`` :doc:`library and its public API </app-dev/bindings-ts/daml-react>`
+     - Stable
+     -
+   * - **DAML Libraries**
+     -
+     -
+   * - :doc:`The DAML Standard Library </daml/stdlib/index>`
+     - Stable
+     -
+   * - :doc:`The DAML Script Library </daml-script/api/index>`
+     - Stable
+     -
+   
+.. _dev-tools:
+
+Developer Tools
+---------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **SDK**
+     -
+     -
+   * - Windows SDK (installer_)
+     - Stable
+     -
+   * - :ref:`Mac SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :ref:`Linux SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :doc:`/tools/assistant` with top level commands
+
+       - ``--help``
+       - ``version``
+       - ``install``
+       - ``uninstall``
+     - Stable
+     -
+   * - ``daml start`` helper command and associated CLI (``daml start --help``)
+     - Stable
+     - 
+   * - ``daml deploy`` :doc:`helper command and associated CLI </deploy/generic_ledger>` (``daml deploy --help``)
+     -
+     -
+   * - **DAML Projects**
+     -
+     -
+   * - ``daml.yaml`` project specification
+     - Stable
+     - 
+   * - Assistant commands ``new``, ``create-daml-app``, and ``init``. Note that the templates created by ``daml new`` and ``create-daml-app`` are considered example code, and are not covered by :ref:`semantic versioning <versioning>`.
+     - Stable
+     -
+   * - **DAML Studio**
+     -
+     -
+   * - VSCode Extension
+     - Stable
+     -
+   * - ``daml studio`` assistant command
+     - Stable
+     -
+   * - **Code Generation**
+     -
+     -
+   * - ``daml codegen`` assistant commands
+     - See :ref:`libraries`.
+     -
+   * - **Dev-Time Runtime Components**
+     -
+     -
+   * - ``daml json-api`, ``daml trigger``, ``daml trigger-service``, and ``daml extractor`` assistant commands.
```

Comment:
dev-time because you use the standalone jars outside of dev-time? This looks pretty confusing to me. None of those are tools that I would consider development-time so if this is really only about the commands, we should spell this out very explicitly. It is also missing the only things that I would consider development time namely `daml navigator` and `daml script` (arguable and I’m fine with using it outside of development but you argued for branding it as development-time so far).

Reply:
I moved these under the "SDK" heading as "Assistant commands to start Runtime Components". Do you agree that that's clearer?

Reply:
Sounds good, thanks!